spring-cloud-stream/src/main/java/org/springframework/cloud/stream/config/BinderProperties.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.stream.config;
+
+import java.util.Properties;
+
+/**
+ * Contains the properties of a binder.
+ *
+ * @author Marius Bogoevici
+ */
+public class BinderProperties {
```

Comment:
I'm all for changing the other `BinderProperties`, but don't want to add more unrelated changes to the PR. So let's change the name of this for now. I'd like to retain the suffix `Properties` for the properties class, since it's more or less what we do for every other typesafe property holder. Would `BinderInstanceProperties` a more suitable compromise? I see the issue about `binderProperties.getProperties()` but I don't know how to call the properties holder better than `properties` , so maybe `BinderInstanceConfiguration` or something.